<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="13203" started-at="2017-12-12T20:50:22Z" finished-at="2017-12-12T20:50:35Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="13203" started-at="2017-12-12T20:50:22Z" finished-at="2017-12-12T20:50:35Z">
      <class name="SeleniumTraining.PHP.PhPCollabTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@614c5515]" name="beforeTest" is-config="true" duration-ms="9674" started-at="2017-12-13T02:20:22Z" finished-at="2017-12-13T02:20:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@614c5515]" name="beforeMethod" is-config="true" duration-ms="3" started-at="2017-12-13T02:20:31Z" finished-at="2017-12-13T02:20:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void SeleniumTraining.PHP.PhPCollabTest.CreateUser() throws java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="CreateUser()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@614c5515]" name="CreateUser" duration-ms="2097" started-at="2017-12-13T02:20:31Z" finished-at="2017-12-13T02:20:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at SeleniumTraining.PHP.PhPCollabTest.CreateUser(PhPCollabTest.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateUser -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@614c5515]" name="afterTest" is-config="true" duration-ms="1405" started-at="2017-12-13T02:20:33Z" finished-at="2017-12-13T02:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="SKIP" signature="CreateProject()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@614c5515]" name="CreateProject" duration-ms="0" started-at="2017-12-13T02:20:33Z" depends-on-methods="SeleniumTraining.PHP.PhPCollabTest.CreateUser" finished-at="2017-12-13T02:20:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateProject -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@614c5515]" name="getResult" is-config="true" duration-ms="0" started-at="2017-12-13T02:20:33Z" finished-at="2017-12-13T02:20:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CreateUser status=FAILURE method=PhPCollabTest.CreateUser()[pri:0, instance:SeleniumTraining.PHP.PhPCollabTest@614c5515] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- SeleniumTraining.PHP.PhPCollabTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
